internal/compiler/passes.rs

```diff
@@ -81,6 +92,9 @@ pub async fn run_passes(
     };
 
     let global_type_registry = type_loader.global_type_registry.clone();
+
+    inject_debug_hooks::inject_debug_hooks(doc, type_loader);
```

Comment:
I think this should go in run_import_passes, if you want to have it on anything else than the main component.

Reply:
But that way this ends up getting called several times, doesn't it?

That would mean I have to avoid instrumenting expressions several times.

Reply:
Yes that's right, but that'd be on different documents. So it would be only called once per file, for the file that is used.

Otherwise, you need to run that pass after  `collect_subcomponents` in order to also mark component declared in other files.

Reply:
I added my pass into `run_import_passes` and added a check to prevent the debug hooks getting added twice.